source/horoscope/pages/landing.html

```diff
@@ -9,7 +9,21 @@
 </head>
 
 <body>
-    <h1 id="landing-title">Horoscope</h1>
+    <div class="header-container">
+        <h1 id="landing-title">Horoscope</h1>
+        <nav class="navbar">
+            <div class="menu-toggle" id="mobile-menu">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <ul class="nav no-search">
+                <li class="nav-item"><a id="daily-horoscope">Daily Horoscope</a></li>
```

Comment:
similarly if we are using the custom way add a `tabindex=0` to all 3 and keypress listeners so it can be interacted with keyboards.